src/main/java/com/lambdaschool/foundation/SeedData.java

```diff
@@ -127,7 +129,7 @@ public void run(String[] args) throws Exception {
                 if(count % 27 == 0) System.out.println((int)(count * 100 / 270) + "%");
 
                 // End of BufferedReader, check size
-                // System.out.println(cityNode.textValue() + ", " + stateNode.textValue() + " data: " + dataArray.size());
+                // System.out.println("data: " + dataArray.size());
```

Comment:
Should we be removing console logs that are no longer needed from production code @numbers0580 ?

Reply:
I had commented out a lot of them in lieu of deletion since I'm not sure if I would use them again to help trace errors if we make further changes. The ones I didn't comment out are there to periodically display an output while it compiles, so we won't falsely think the Back End had frozen. To get this to work, I had to make hundreds of calls to the DS API.